content/c-sharp/concepts/data-types/data-types.md

```diff
@@ -0,0 +1,67 @@
+---
+Title: 'Data Types'
+Description: 'C-sharp supports many data types that represent the size and kind of values being stored in memory.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Data Types'
+  - 'Integers'
+  - 'Strings'
+  - 'Booleans'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The two main data types in C# are value ane reference types.
+
+## Value Types
+
+Value types are data types that are built in to C#. The available types and their memory sizes are as follows:
+
+| Data Type | Memory Size  |
```

Comment:
Since we're briefly going over all the types in this entry, can we create an additional column here for a short description about what each type represents?